intl/msg_hash_us.h

```diff
@@ -3485,6 +3641,10 @@ MSG_HASH(
    MENU_ENUM_SUBLABEL_CHECK_FOR_MISSING_FIRMWARE,
    "Check if all the required firmware is present before attempting to load content."
    )
+MSG_HASH(
+   MENU_ENUM_LABEL_HELP_CHECK_FOR_MISSING_FIRMWARE,
+   "Some cores might need firmware or bios files.\nIf this option is disabled, it will try to load even if such firmware is missing. !!!"
```

Comment:
Bad inheritance here: those three exclamation marks should be removed or replaced with something that makes more sense.

Reply:
Noted, will fix in phase 2 PR. I think I added the exclamation marks to remind me to improve it, but forgot it eventually :/

Reply:
Suggestion:
"Some cores might need firmware or bios files.\nIf this option is enabled, RetroArch will not allow starting the core if any mandatory firmware items are missing."
The problem is, that this function works unreliably as I tested now... But it is a separate issue, that should be fixed.

Reply:
This and other changes can now be seen at https://github.com/zoltanvb/RetroArch/commit/633caf83cf24c3492e6d0d736618219fa3b7e6c8